### PR TITLE
Decryption message

### PR DESCRIPTION
**Pull request by adel-gu**
In this PR I accomplished solving the decoding of Morse message.

### CODE REVIEW

File: decrypt.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      decoded_seq += get_char(c)
+    end
+
+    # when the loop ends this means creating a space between words.
+    decoded_seq += ' '
+  end
+
+  # return the result
+  decoded_seq
+end
```
Comment (julie-ify) [inline decrypt.rb]: - [x] You currently have only two methods, meanwhile, you are required to have three methods:
  - The first method decodes A SINGLE CHARACTER
  - The second method decodes A WORD using the first method
  - The last method decodes A SENTENCE using the second method.

  Kindly refactor for that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @adel-gu,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: decrypt.rb (hunk 1 above)
Comment (adel-gu) [reply decrypt.rb]: Hi @julie-ify, I did so, thank you very much for the note 👏👏👏

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @adel-gu , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




- [OPTIONAL] I noticed that the MIT link in your README is broken, so I suggest to fix it 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**